spec/views/followups/_followup.html.erb_spec.rb

```diff
@@ -97,4 +97,25 @@
     end
 
   end
+
+  describe 'displaying followup contact options' do
+
+    context 'with an incoming message' do
+
+      let(:incoming) do
+        FactoryBot.create(:plain_incoming_message, info_request: info_request)
+      end
+
+      it 'does not offer the same contact option twice' do
```

Comment:
I don't think the template is responsible for deciding this? At least, I can't work out where it does deduplication – that happens in the model layer as of this commit, so I don't think a view spec is necessary here?

Reply:
True - it just iterates over what it's given